api/types/license.go

```diff
@@ -358,6 +376,8 @@ type LicenseSpecV3 struct {
 	SupportsDatabaseAccess Bool `json:"db,omitempty"`
 	// SupportsDesktopAccess turns desktop access on or off
 	SupportsDesktopAccess Bool `json:"desktop,omitempty"`
+	// SupportsSAMLIdP turns SAMLIdP access on or off
+	SupportsSAMLIdP Bool `json:"saml_idp,omitempty"` //nolint:revive // Because we want this to be IdP.
```

Comment:
I wonder if we can set a global ignore/allow for `IdP` ?

Reply:
I looked into it. Basically the `revive` linter seems not to allow mixed case even if you try to relax the restrictions AFAICT.